### DISCUSSION TITLE
Can I package the Torch-GPU with PyInstaller

### DISCUSSION THREAD

**Discussion by zhanghongyong123456** [pyinstaller/pyinstaller#5904] [Help]
I have a question. I have a Python project that is implemented on Torch-GPU. Can I package it with PyInstaller and run it independently on other computers, or what do I need to configure to run it on other computers.

I have tried a demo, which can be used normally on this machine, but this bug appears on other computers, I hope to get your reply
![103](https://user-images.githubusercontent.com/48466610/121663902-7be12a00-cad9-11eb-902e-8ac722525abf.png)

**Comment by rokm** [comment]
You probably need to install [Visual C++ Redistributable](https://aka.ms/vs/16/release/vc_redist.x64.exe) on the target computers. 

Or, try using PyInstaller  `develop` branch, which has improved collection of those VC runtime DLLs.

**Reply by zhanghongyong123456** [reply]
I have tried both methods and still got the same error. Could you please give me another method? Thank you very much

**Reply by rokm** [reply]
Then you're on your own; try opening the `caffe2_detectron_ops_gpu.dll` with  [dependency walker](https://www.dependencywalker.com/) on the target computer and see what DLL it is missing.

**Comment by zhanghongyong123456** [comment]
I have tried both methods and still got the same error. Could you please give me another method? Thank you very much
